### PR TITLE
System integration test

### PR DESCRIPTION
**Pull request by srudaa**

### CODE REVIEW

**Comment by rafaelschlatter** [comment]
Nice documentation! This would definitely be helpful on all modules & connectors. Ill add to the ones im working on.

File: Source/IdentityMapper.csproj [hunk 1]
```diff
@@ -9,10 +9,10 @@
   <ItemGroup>
     <PackageReference Include="Autofac" Version="6.1.0" />
     <PackageReference Include="Serilog" Version="2.10.0" />
-    <PackageReference Include="RaaLabs.Edge" Version="1.0.0" />
-    <PackageReference Include="RaaLabs.Edge.Modules.EventHandling" Version="1.0.0" />
-    <PackageReference Include="RaaLabs.Edge.Modules.EdgeHub" Version="1.0.0" />
-    <PackageReference Include="RaaLabs.Edge.Modules.Configuration" Version="1.0.0" />
+    <PackageReference Include="RaaLabs.Edge" Version="1.0.1" />
```
Comment (thotokraa) [inline Source/IdentityMapper.csproj:12]: The latest version is 1.0.4. Please update.

File: Source/IdentityMapperHandler.cs [hunk 2]
```diff
@@ -4,10 +4,12 @@
 using RaaLabs.IdentityMapper.events;
 using Serilog;
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 namespace RaaLabs.IdentityMapper
 {
-    class IdentityMapperHandler : IConsumeEvent<EdgeHubDataPointReceived>, IProduceEvent<EdgeHubDataPointRemapped>
+    [ExcludeFromCodeCoverage] // Tested in system integration test
```
Comment (thotokraa) [inline Source/IdentityMapperHandler.cs:11]: This handler should be tested, now that we have solved how to do that.

File: Specifications/Features/SystemIntegration.Specs.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Feature: System integration
+    Put feature description here
```
Comment (thotokraa) [inline Specifications/Features/SystemIntegration.Specs.feature:2]: Not really that descriptive :P

File: Specifications/Steps/Steps.cs [hunk 4]
```diff
@@ -3,8 +3,6 @@
  *  Licensed under the MIT License. See LICENSE in the project root for license information.
```
Comment (thotokraa) [inline Specifications/Steps/Steps.cs:3]: This class needs a better, more descriptive name. I suggest TimeSeriesMapperSteps.

File: Specifications/Steps/TagMappingSteps.cs [hunk 5]
```diff
@@ -5,7 +5,7 @@
 using TechTalk.SpecFlow;
 using System.Linq;
 using BoDi;
-using System;
+using Serilog;
 
 namespace RaaLabs.IdentityMapper.Specs.StepDefinitions
 {
```
Comment (thotokraa) [inline Specifications/Steps/TagMappingSteps.cs:11]: I can't see that this class is being used in a .feature file. Can it be removed?

**Review by thotokraa — CHANGES_REQUESTED** [review]
Please fix the issues commented, plus the code coverage warnings.

**Review by thotokraa — APPROVED** [review]

**Pull request merged**